go.mod

```diff
@@ -50,8 +50,8 @@ require (
 	github.com/morikuni/aec v1.0.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0-rc3
-	github.com/opencontainers/runc v1.1.6
-	github.com/opencontainers/runtime-spec v1.1.0-rc.1
+	github.com/opencontainers/runc v1.1.7
```

Comment:
Might want follow-up to bump runc in Dockerfile

Reply:
done in https://github.com/moby/buildkit/pull/3886